lib/platform.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+export function is_nodejs() {
```

Comment:
I'm pretty sure there's an NPM package for this <sup><sub>/s</sub></sup>

Reply:
![THANKS I HATE IT](https://res.cloudinary.com/teepublic/image/private/s--_yGBeyxY--/t_Preview/b_rgb:262c3a,c_limit,f_jpg,h_630,q_90,w_630/v1523656740/production/designs/2588180_0.jpg)

Yeah, I know there is, I didn't want to take a dependency for this little bit of code.